.github/workflows/linux.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           SCONS_CACHE: ${{github.workspace}}/.scons_cache/
         run: |
           PATH=${GITHUB_WORKSPACE}/x86_64-godot-linux-gnu_sdk-buildroot/bin:$PATH
-          scons -j2 verbose=yes warnings=all werror=yes platform=linuxbsd tests=no dev_build=${{matrix.dev_build}} dev_mode=yes target=${{matrix.target}} production=${{matrix.production}} float=${{matrix.float}}
+          scons -j2 verbose=yes warnings=extra werror=yes platform=linuxbsd tests=no dev_build=${{matrix.dev_build}} dev_mode=yes target=${{matrix.target}} production=${{matrix.production}} float=${{matrix.float}}
```

Comment:
What does the change from all to extra entails? Is it even more warnings or just an "extra warning" subset of those?

Reply:
https://github.com/the-mirror-megaverse/mirror_voxel/pull/20#discussion_r1003259338 (next comment) answered it.